### PR TITLE
[BE-04] Add price field

### PR DESCRIPTION
**Pull request by Enriquee98**
https://trello.com/c/vJ7qpFRJ

AC:
1. price field may be set during create or update a record
2. price field is returned in #show response.
4. price field returns ‘unknow’ if value is empty.

### CODE REVIEW

**Comment by plokijro** [comment]
Resolve conflicts pls

File: app/controllers/api/v1/recipes_controller.rb [hunk 1]
```diff
@@ -38,7 +38,7 @@ def destroy
       private
 
       def recipe_params
-        params.require(:recipe).permit(%i[name content level cooking_time video_link])
+        params.require(:recipe).permit(%i[name content price level cooking_time video_link])
```
Comment (plokijro) [inline app/controllers/api/v1/recipes_controller.rb:41]: Add Controller specs - create and update

File: spec/models/recipe_spec.rb [hunk 2]
```diff
@@ -16,4 +16,9 @@
     it { is_expected.to allow_value('', nil).for(:video_link) }
     it { is_expected.to validate_url_of(:video_link) }
   end
+
+  describe 'price should be decimal + can be nil' do
+    it { is_expected.to validate_numericality_of(:price).is_greater_than_or_equal_to(0) }
+    it { is_expected.to allow_value(nil).for(:price) }
+  end
```
Comment (grabomarcin) [inline spec/models/recipe_spec.rb]: This describe is unnecessary. You can place that in `validations`.

File: spec/serializers/recipe_show_serializer_spec.rb [hunk 3]
```diff
@@ -5,6 +5,16 @@
...
+  describe '#price' do
+    context 'with price' do
+      it { is_expected.to include(price: recipe.price) }
+    end
+
+    context 'without price' do
+      let!(:recipe) { create :recipe, price: nil }
+
+      it { is_expected.to include(price: described_class::NIL_RESULT) }
+
```
Comment (grabomarcin) [inline spec/serializers/recipe_show_serializer_spec.rb]: You forgot to close context block

**Review by plokijro — APPROVED** [review]

**Pull request merged**